qubekit/molecules/ligand.py

```diff
@@ -1036,6 +1036,21 @@ class Ligand(Molecule):
     wbo: Optional[Array[float]] = Field(
         None, description="The WBO matrix calculated at the QM optimised geometry."
     )
+    fragments: Optional[List["Ligand"]] = Field(
+        None,
+        description="Fragments in the molecule with the bonds around which the fragments were built.",
+    )
+    bond_indices: Tuple[int, int] = Field(
+        None,
+        description="The map indices of the atoms involved in the bond (in the parent molecule) that the "
+        "fragment was built around.",
+    )
+    atom_idx_map: Optional[Dict[int, int]] = Field(
```

Comment:
Do we need this still? I think if we use the mapping index in the bond indices they should have the same map index in the parent and the fragment?